### PR TITLE
Parallel tests

### PR DESCRIPTION
**Pull request by odisseus**
These changes allow the tests to be executed in parallel.

### CODE REVIEW

**Comment by liosedhel** [comment]
Thanks @odisseus for this PR :) Out of curiosity, what was the main motivation behind this PR - were those tests annoyingly slow or something else pushed you to work on this? :)

**Comment by odisseus** [comment]
Fury doesn't have a flag that forces the Scalatest runner to use a single thread.

File: src/test/scala/org/virtuslab/beholder/BaseTest.scala [hunk 1]
```diff
@@ -10,53 +10,61 @@ import org.virtuslab.beholder.repositories.{ MachineRepository, UserMachinesRepo
...
-    Play.stop(app)
-    super.afterEach()
+  def withFixture(test: OneArgTest): Outcome = {
+    val dbSuffix = scala.util.Random.nextInt().toHexString
+    var fakeApp: Option[Application] = None
+    try {
+      val theFixture = createFixture(dbSuffix)
+      import theFixture.unicorn.profile.api._
+      fakeApp = Some(theFixture.app)
+      val DB = DatabaseConfigProvider.get(theFixture.app).db
```
[the remark this answers is not in the document]
Comment (odisseus) [reply src/test/scala/org/virtuslab/beholder/BaseTest.scala]: Ok

**Comment by liosedhel** [comment]
LGTM

**Pull request merged**